payment/src/test/java/org/killbill/billing/payment/logging/TestLoggingHelper.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2014-2015 Groupon, Inc
+ * Copyright 2014-2015 The Billing Project, LLC
+ *
+ * The Billing Project licenses this file to you under the Apache License, version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at:
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.killbill.billing.payment.logging;
+
+import java.lang.reflect.Field;
+import java.util.Map;
+import java.util.concurrent.Callable;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.impl.SimpleLoggerFactory;
+import org.slf4j.impl.StaticLoggerBinder;
+
+/**
+ * Created by holger on 10/14/15.
+ */
+public class TestLoggingHelper {
```

Comment:
I understand that. But this class isn't a test class. It is called from test classes that already inherit from PaymentTestSuiteNoDB, therefore everything that should be in place is already in place. I have adjusted it to your wish, but I am just not convinced.